src/pyclaw/sharpclaw/solver.py

```diff
@@ -289,6 +306,8 @@ def step(self,solution,take_one_step,tstart,tend):
         if self.accept_step == True:
             self.cfl.set_global_max(0.)
             self.dq_dt = self.dq(state) / self.dt
+            if 'DW' in self.time_integrator:
```

Comment:
This has to do with our convention for the names of different time integrators.
I now check if `self.time_integrator` starts with 'DW', since I assume that we follow this notation for all downwind methods.